src/App.css

```diff
@@ -3,3 +3,9 @@
   color: white;
   background-color: #243e5d;
 }
+
+.dg_news_date-author p span:first-child {
+  border-right: 2px solid grey;
```

Comment:
Should grey be `#292929` as per the [XD Doc](https://xd.adobe.com/view/5e59f202-fe25-433e-4697-230f34045cd2-2868/screen/3721b95f-8faa-4c08-ab0c-05bc958c5ac2/Application-Newsroom)?

Reply:
I dont know if that hex value matches the same color as $gray-dark in the component library but I changed it anyway since it should be close enough. Also reduced to 1px from 2px on width since that matches XD also.